### PR TITLE
Api documentation update

### PR DESCRIPTION
**Pull request by davidjovanovic**

### CODE REVIEW

File: src/main/java/it/eng/idsa/businesslogic/configuration/WebConfig.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package it.eng.idsa.businesslogic.configuration;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.CorsRegistry;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+@Configuration
+@EnableWebMvc
+public class WebConfig implements WebMvcConfigurer {
```
Comment (IgorBalog-Eng) [inline src/main/java/it/eng/idsa/businesslogic/configuration/WebConfig.java:10]: Can this one be replaced in SecurityConfig.java with .cors()?
https://www.baeldung.com/spring-cors#cors-with-spring-security

**Review by IgorBalog-Eng — COMMENTED** [review]
Please address comments

File: pom.xml [hunk 2]
```diff
@@ -172,6 +167,13 @@
 			<artifactId>camel-spring-boot-starter</artifactId>
 			<version>${camel.version}</version>
 		</dependency>
+		<!--  CHECK IF THIS IS NEEDED -->
```
[the remark this answers is not in the document]
Comment (davidjovanovic) [reply pom.xml:170]: this comment? Leftover from who now's when 😄

**Review by IgorBalog-Eng — COMMENTED** [review]
Can you also add information regarding Swagger UI in ECC readme?

File: src/main/java/it/eng/idsa/businesslogic/configuration/WebConfig.java (hunk 1 above)
Comment (davidjovanovic) [reply src/main/java/it/eng/idsa/businesslogic/configuration/WebConfig.java:10]: Didn't know about that one. Will do, thanks!

**Comment by davidjovanovic** [comment]
> 

Sure, no problem

**Review by IgorBalog-Eng — APPROVED** [review]

**Pull request merged**